cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -311,6 +311,81 @@
 }
 
 /* topdoc
+  name: Adds block background that bleeds off page 
+  family: cf-layout
+  notes:
+    - ".block__bg-extend-right requires that overflow-x be set to 
+       hidden on a parent element."
+  tags:
+    - cf-layout
+*/
+
+.block {
```

Comment:
Ok. Would it be better if I scope the changes locally for now and then create a future PR where we can test them as block modifiers?